docs/Concepts/Privacy/Privacy-Overview.md

```diff
@@ -27,28 +27,51 @@ participants. Other participants cannot access the transaction content or list o
 ## Private transaction manager
 
 Besu uses a private transaction manager, [Tessera](https://docs.tessera.consensys.net/), to implement
-privacy. Each Besu node sending or receiving private transactions requires an associated Tessera
-node.
+privacy.
+Each Besu node sending or receiving [private transactions](Private-Transactions.md) requires an
+associated Tessera node.
 
 ![Tessera Nodes](../../images/TesseraNodes.png)
 
 Private transactions pass from the Besu node to the associated Tessera node. The Tessera node
 encrypts and directly distributes (that is, point-to-point) the private transaction to the Tessera
 nodes participating in the transaction.
 
-By default, each participant in a privacy network uses its own Besu and Tessera node.
+By default, each participant in a privacy-enabled network uses its own Besu and Tessera node.
 [Multi-tenancy](Multi-Tenancy.md) allows more than one participant to use the same Besu and Tessera
 node.
 
 !!! tip
 
     Private Transaction Managers are also known as Enclaves.
 
+## Privacy-enabled networks
+
+When enabling privacy in a [private network](../../HowTo/Get-Started/System-Requirements/System-Requirements-Private.md),
+there's an assumed level of trust among the node operators, since all are members of the private
+network.
+
+Because gas isn't included in private transactions, inefficient contracts, either accidentally or
```

Comment:
nit:"Inefficient contracts deployed accidentally or deliberately can cause performance issues because gas isn't required."

Perhaps in an important admonition?